gebrueder-weiss-woocommerce.php

```diff
@@ -45,12 +54,13 @@ function GbWeiss()
     // Initialize the plugin here.
     $plugin = GbWeiss();
 
+
     /**
      * Validate client credentials
      * this will later be shifted to the corresponding save / validation workflows
      */
     $authenticationClient = new OAuthAuthenticator(new GuzzleHttp\Client());
-    $authenticationClient->setAuthenticationEndpoint('http://18019fdce8ff:8887/token');
+    $authenticationClient->setAuthenticationEndpoint($_ENV['GEBRUEDER_WEISS_OAUTH_URL']);
```

Comment:
I would use the `getenv` function for accessing the environment variable and provide the production endpoint as a fallback, since the environment variable will not be set if some random customer installs the plugin. We can use it to configure the plugin to use mock/testing endpoints.

```
$authenticationClient->setAuthenticationEndpoint(getenv('GEBRUEDER_WEISS_OAUTH_URL') ?: "INSERT_PRODUCTION_URL_HERE");
```